### PR TITLE
Add option to freeze BatchNorm layers.

### PR DESCRIPTION
**Pull request by hgaiser**
This PR adds the ability to freeze BatchNorm layers, which is often done when finetuning algorithms (such as Faster RCNN). The freezing is implemented in a custom BatchNorm layer which calls Keras' BatchNorm function, while setting the two different `training` and `trainable` flags. The reason for a custom layer is because the alternative would look something like this:

```
    layer = keras.layers.BatchNormalization()
    layer.trainable = not freeze_bn
    output = layer(input, training=(not freeze_bn))
```

The custom layer changes this to:

```
output = keras_resnet.layers.BatchNormalization(freeze=freeze_bn)(input)
```

which saves lines and complexity.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,2 +1,7 @@
+# ignore:
+# E501 line too long (98 > 79 characters)
+[tool:pytest]
+pep8ignore = E501
```
Comment (0x00b1) [inline setup.cfg]: Thanks

**Comment by 0x00b1** [comment]
Awesome. Thanks, @hgaiser.
